### PR TITLE
Removes Deployment to Treescale File

### PR DESCRIPTION
**Pull request by allen-tran**
image

### CODE REVIEW

**Review by evanugarte — APPROVED** [review]
Still have some random changes in package-lock. To fix, do:

```sh
git checkout origin/dev package-lock.json
auc
```

File: package-lock.json [hunk 1]
```diff
@@ -39821,4 +39821,4 @@
       "version": "0.1.0"
     }
   }
-}
+}
```
Comment (evanugarte) [inline package-lock.json:39824]:
```suggestion
}

```

**Comment by imdominicreed** [comment]
😳

**Review by imdominicreed — APPROVED** [review]
very cute code! 😳 🥵

**Comment by allen-tran** [comment]
> very cute code! 😳 🥵

let's keep this convo between us ;)

**Pull request merged**